readthedocs_build/config/test_find.py

```diff
@@ -75,3 +75,18 @@ def test_find_multiple_files(tmpdir):
         str(tmpdir.join('first', 'readthedocs.yml')),
         str(tmpdir.join('third', 'readthedocs.yml')),
     ]
+
+
+def test_find_unicode_path(tmpdir):
```

Comment:
This test can also be written to fail, using the `pytest.xfail` decorator. Instead of writing a test that will fail once we have the bug fixed, we can write a passing case. Our tests will technically pass, but there will be an expected failure on the test run.